runSensorNode.py

```diff
@@ -1,13 +1,58 @@
 #!/usr/bin/python
 import os
+import appdirs
+import logging
+import logging.handlers
+import pathlib
 
 from SensorNode.node import createSensorNode
 
-if __name__ == '__main__':
+
+def main():
+    if os.getuid() == 0:
+        log_dest = os.path.join('var', 'log', 'asm_sensor_node.log')
+    else:
+        log_dir = appdirs.user_log_dir('ASM Sensor Node')
```

Comment:
Change this to `ASMSensorNode` to be consistent with the server code, and to facilitate entry into the terminal